tests/app.test.js

```diff
@@ -0,0 +1,23 @@
+import chai, { should } from 'chai';
+import chaiHttp from 'chai-http';
+
+import app from '../src';
+
+chai.use(chaiHttp);
+should();
+
+describe('Root Route', () => {
+  it('should pass on GET/', async () => {
+    const response = await chai.request(app).get('/');
+    response.should.have.status(200);
+    response.body.should.be.a('object');
+  });
+});
+
+describe('Error Messages', () => {
+  it('should display 404 error on invalid route', async () => {
+    const res = await chai.request(app).get('/invalid/route');
+    res.should.have.status(404);
```

Comment:
We have to decide to use either should or expect. This will help us achieve consistency. Also, please leave a new line between 19 and 20.